readthedocs/organizations/views/private.py

```diff
@@ -151,7 +151,9 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
-class DeleteOrganizationOwner(PrivateViewMixin, OrganizationOwnerView, DeleteView):
+class DeleteOrganizationOwner(
+    PrivateViewMixin, OrganizationOwnerView, DeleteViewWithMessage
```

Comment:
Here for example, we already are showing a message manually on a successful deletion. By inheriting from `DeleteViewWithMessage`, we are now showing two messages on success, and one on error.